packages/web/babel.ts

```diff
@@ -16,6 +17,9 @@ interface NxReactBabelPresetOptions {
 }
 
 module.exports = function (api: any, options: NxReactBabelPresetOptions = {}) {
+  console.warn(
```

Comment:
suggestion: maybe use `logger` from `nrwl/devkit` (which uses `chalk` under the hood) for better warning mesage